modules/mod_ginger_collection/templates/collection/metadata/geo.tpl

```diff
@@ -6,7 +6,7 @@
         </h6>
         <div class="adlib-object__meta__data">
             {% include "list/list-uri-labels.tpl" items=places %}
-            {% include "map/map.tpl" result=places %}
+            {% include "map/map.tpl" result=places height=200 %}
```

Comment:
Doesn’t this change the height from 300 to 200px?

Reply:
True, solved.